kolibri/plugins/oidc_provider_plugin/api_urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import include
+from django.conf.urls import url
+from rest_framework import routers
+
+router = routers.SimpleRouter()
```

Comment:
Looks like this is doing nothing, as we are not registering any endpoints against the router.